deploy/aws-account-operator-template.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: aws-account-operator-template
+
+
+objects:
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: account-reader
+      namespace: aws-account-operator
+    rules:
+      - apiGroups:
+          - aws.managed.openshift.io
+        resources:
+          - accounts
+        verbs:
+          - '*'
```

Comment:
Since we're here, can we make this be actual reader-ish perms? If not, at least change the names.

Reply:
i have updated the permission to 'get' and 'list'